doc/zephyr.doxyfile.in

```diff
@@ -759,7 +759,7 @@ INPUT                  = @ZEPHYR_BASE@/include/ \
                          @ZEPHYR_BASE@/include/arch/arm/cortex_m \
                          @ZEPHYR_BASE@/include/arch/nios2/ \
                          @ZEPHYR_BASE@/lib/libc/minimal/include/ \
-                         @ZEPHYR_BASE@/include/net/coap.h \
+                         @ZEPHYR_BASE@/include/net/coap_sock.h \
```

Comment:
Technically, since ``RECURSIVE = YES`` is specified, all the subdirectories of ``/include`` are scanned so we don't need to have any of these directories or files specified within the ``/include`` folder, so this change isn't needed.  (I should submit a separate PR to remove all these folder/file names that would be included by mentioning ``/include``.)

Reply:
> I should submit a separate PR to remove all these folder/file names that would be included by mentioning `/include`

ok